src/components/App/App.tsx

```diff
@@ -143,7 +143,7 @@ const App: FC<Props> = ({
           />
         </Section>
       )}
-      {activeMenuItemId === 'chartElements' && (
+      {activeMenuItemId === 'chart_elements' && (
```

Comment:
what do you think about wrapping `activeMenuItemId` with type for safety?

Reply:
Good idea, will do